yt_dlp/extractor/extractors.py

```diff
@@ -1,28 +1,35 @@
-import contextlib
+import inspect
 import os
 
-from ..plugins import load_plugins
+from ..globals import LAZY_EXTRACTORS
+from ..globals import extractors as _extractors_context
 
-# NB: Must be before other imports so that plugins can be correctly injected
-_PLUGIN_CLASSES = load_plugins('extractor', 'IE')
-
-_LAZY_LOADER = False
+_CLASS_LOOKUP = None
 if not os.environ.get('YTDLP_NO_LAZY_EXTRACTORS'):
-    with contextlib.suppress(ImportError):
-        from .lazy_extractors import *  # noqa: F403
-        from .lazy_extractors import _ALL_CLASSES
-        _LAZY_LOADER = True
+    try:
+        from .lazy_extractors import _CLASS_LOOKUP
+        LAZY_EXTRACTORS.value = True
```

Comment:
Can you explain this to me? When will importing `_CLASS_LOOKUP` from `.lazy_extractors` *not* raise?

No matter what I try I am getting "Lazy loading extractors is forcibly disabled" in debug

Reply:
looks like we missed implementing `_CLASS_LOOKUP` in `make_lazy_extractors.py` script. should be fixed now